mteb/tasks/BitextMining/multilingual/RomaTalesBitextMining.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskBitextMining
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class RomaTalesBitextMining(AbsTaskBitextMining):
```

Comment:
This class should inherit `CrossLingualTask` class, see example [here](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/BUCCBitextMining.py)